### PR TITLE
Configuration file

### PR DESCRIPTION
**Pull request by keveri**
Configuration file stuff

related to #6

### CODE REVIEW

**Comment by keveri** [comment]
Needs to take parameter `user.conf` which should override `application.conf`

**Comment by keveri** [comment]
Closes #6

File: src/main/scala/neljas/Main.scala [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
 
+import com.typesafe.config._
 import com.typesafe.scalalogging.{LazyLogging, Logger}
 import org.http4s.server.blaze.BlazeBuilder
 
+import neljas.conf.Conf
+
 object Main extends App with LazyLogging {
-  val port = Option(System.getenv("PORT")).map(_.toInt).getOrElse(8080)
+  if (Main.args.length < 1) {
```
Comment (jkpl) [inline src/main/scala/neljas/Main.scala]: `Main.args` can be replaced with just `args` when used inside `Main`

File: src/main/scala/neljas/conf/Conf.scala [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    val custom = ConfigFactory.parseFile(file)
+    Settings(custom.withFallback(default))
+  }
+}
+
+final case class Settings(conf: Config) {
+  val port    = conf.getInt("port")
+  val pdfPath = conf.getString("pdf_path")
+
+  val smtpHost     = conf.getString("smtp.host")
```
Comment (jkpl) [inline src/main/scala/neljas/conf/Conf.scala]: You can get the `smtp` part of the config like this: `val smtpConf = conf.getConfig("smtp")`

You can then get values from config like this: `val smtpHost = smtpConf.getString("host")`

**Pull request merged**